packages/xstate-graph/src/graph.ts

```diff
@@ -62,9 +62,11 @@ export function getChildren(stateNode: StateNode): StateNode[] {
   return children;
 }
 
-export function serializeState<TContext>(state: State<TContext, any>): string {
+export function serializeState<TContext extends MachineContext>(
+  state: State<TContext, any>
+): string {
   const { value, context } = state;
-  return context === undefined
+  return context === undefined || Object.keys(context).length === 0
```

Comment:
`context` should never be undefined noow, right? :)
```suggestion
  return Object.keys(context).length === 0
```